nltk/test/unit/test_tokenize.py

```diff
@@ -54,6 +54,77 @@ def test_tweet_tokenizer(self):
         ]
         assert tokens == expected
 
+    @pytest.mark.parametrize(
+        "match_phone_numbers,test_input,expected",
```

Comment:
nit: with parametrize, i like putting a space between args, like `"match_phone_numbers, test_input, expected"`. feels easier to read!

also i wonder if you could squeeze some of these test cases into one-liners. (okay if not haha)

Reply:
I've placed a space between the args, and I've shrunk down the test cases a bit, to avoid needing to duplicate the `test_input`. Perhaps that's what you were after with the one-liner stuff? I'm not quite sure.

Reply:
Yup, exactly what I meant! Also nice, our first instance of typing in nltk 👍